src/uct/ib/rc/accel/rc_mlx5_common.h

```diff
@@ -155,15 +155,14 @@ typedef struct uct_rc_mlx5_srq_op {
 
 
 /* Command QP work-queue. All tag matching list operations are posted on it. */
-typedef struct uct_rc_mlx5_cmd_wq {
+typedef struct uct_rc_mlx5_cmd_qp {
```

Comment:
why need to rename this type? seems like it's still cmd_wq ?

Reply:
https://github.com/openucx/ucx/pull/3439#discussion_r272834237

Reply:
right. i was wrong with the original comment https://github.com/openucx/ucx/pull/3439#discussion_r272834237, now i've noticed xx_wq always contains xx_qp. so it's ok to keep 'wq' name.